lib/px/utils/pxclient.lua

```diff
@@ -86,6 +86,11 @@ function M.load(config_file)
         if event_type == 'page_requested' and not px_config.send_page_requested_activity then
             return
         end
+			
+	if px_config.additional_activity_handler ~= nil then
+		px_logger.debug("additional_activity_handler was triggered");
+		px_config.additional_activity_handler(event_type, ngx.ctx, details)
```

Comment:
Its a function that the client should implement, thats why its additional
see in [pxconfig](https://github.com/PerimeterX/perimeterx-nginx-plugin/pull/50/files/fc5abe8f959277deb5850cc6b8ee9e52164af948#diff-150cb6ed60d368445c00b5b980c38d48R25)